Fix this warning:

```bash
$ shellcheck test.sh                         

In test.sh line 51:
        cd "$testsh_pwd"
        ^--------------^ SC2164 (warning): Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

Did you mean: 
        cd "$testsh_pwd" || exit

For more information:
  https://www.shellcheck.net/wiki/SC2164 -- Use 'cd ... || exit' or 'cd ... |...
```

--- a/tests/0001-plan-files-enabled.t
+++ b/tests/0001-plan-files-enabled.t
@@ -8,7 +8,7 @@ _t_plan_files_enabled () {
     pwd
     cp -a "$testsh_pwd/tests/null-resource-hello-world.tfd" "$tmp/"
     _check_and_delete_provider_files "$tmp/null-resource-hello-world.tfd" "$TF_VER"
-    cd "$tmp"/null-resource-hello-world.tfd || exit 1
+    cd "$tmp"/null-resource-hello-world.tfd || return 1
     if      $testsh_pwd/terraformsh plan
     then
 
@@ -38,7 +38,7 @@ _t_plan_files_enabled_cd_dir () {
     cp -a "$testsh_pwd/tests/null-resource-hello-world.tfd" "$tmp/"
     _check_and_delete_provider_files "$tmp/null-resource-hello-world.tfd" "$TF_VER"
     mkdir -p "$tmp/rundir"
-    cd "$tmp"/rundir || exit 1
+    cd "$tmp"/rundir || return 1
     if      $testsh_pwd/terraformsh -C "$tmp/null-resource-hello-world.tfd" plan
     then
 

--- a/tests/0002-plan-files-disabled.t
+++ b/tests/0002-plan-files-disabled.t
@@ -45,7 +45,7 @@ _t_plan_files_disabled_cd_dir () {
 
     rm -rf "$tmp"/rundir
     mkdir -p "$tmp/rundir"
-    cd "$tmp"/rundir
+    cd "$tmp"/rundir || return 1
 
     if      $testsh_pwd/terraformsh -C "$tmp/null-resource-hello-world.tfd" -P plan
     then
@@ -81,7 +81,7 @@ _t_plan_files_disabled_destroy () {
     rm -rf "$tmp"/null-resource-hello-world.tfd
     cp -a "$testsh_pwd/tests/null-resource-hello-world.tfd" "$tmp/"
     _check_and_delete_provider_files "$tmp/null-resource-hello-world.tfd" "$TF_VER"
-    cd "$tmp"/null-resource-hello-world.tfd
+    cd "$tmp"/null-resource-hello-world.tfd || return 1
 
     set -e
 
@@ -125,7 +125,7 @@ _t_plan_files_disabled_apply_tfvars () {
     rm -rf "$tmp"/local-file-hello-world.tfd
     cp -a "$testsh_pwd/tests/local-file-hello-world.tfd" "$tmp/"
     _check_and_delete_provider_files "$tmp/local-file-hello-world.tfd" "$TF_VER"
-    cd "$tmp"/local-file-hello-world.tfd
+    cd "$tmp"/local-file-hello-world.tfd || return 1
 
     cat >terraform.sh.tfvars <<EOTFFILE1
 insert-value = "(this is from the tfvars file)"

--- a/tests/0006-apply-no-tfvars.t
+++ b/tests/0006-apply-no-tfvars.t
@@ -8,7 +8,7 @@ _t_apply_no_tfvars () {
 
     rm -rf "$tmp"/local-file-hello-world.tfd
     cp -a "$testsh_pwd/tests/local-file-hello-world.tfd" "$tmp/"
-    cd "$tmp"/local-file-hello-world.tfd || exit 1
+    cd "$tmp"/local-file-hello-world.tfd || return 1
 
     if      $testsh_pwd/terraformsh plan apply
     then

